src/transformers/utils/fx.py

```diff
@@ -22,16 +24,93 @@
     MODEL_FOR_SEQ_TO_SEQ_CAUSAL_LM_MAPPING,
     MODEL_FOR_SEQUENCE_CLASSIFICATION_MAPPING,
     MODEL_FOR_TOKEN_CLASSIFICATION_MAPPING,
+    MODEL_MAPPING,
     GPT2DoubleHeadsModel,
+    PretrainedConfig,
     PreTrainedModel,
     logging,
 )
+from ..models.albert import AlbertConfig
 from ..models.auto import get_values
+from ..models.bert import BertConfig
+from ..models.distilbert import DistilBertConfig
+from ..models.electra import ElectraConfig
+from ..models.gpt2 import GPT2Config
+from ..models.gpt_neo import GPTNeoConfig
+from ..models.gptj import GPTJConfig
+from ..models.megatron_bert import MegatronBertConfig
+from ..models.mobilebert import MobileBertConfig
+from ..models.t5 import T5Config
```

Comment:
We have worked really hard to decouple files of the library, so I am very much against importing all configs in this file. If you really need the config objects, the `CONFIG_MAPPING` will map a model type to its config, so you should use that instead.

Reply:
Actually if I understand correctly the MODEL_...._MAPPINGs are mapping a config class to a model class so that is the reason I was importing the config classes.
But thanks to your review I learned about CONFIG_MAPPING which maps model names to config classes. 
So doing `model name -> model config class -> model class` solves the issue with very small changes in the code, thanks!